### PR TITLE
Jamstack 2021 queries

### PR DESCRIPTION
**Pull request by denar90**
Queries for #2156 
These are most of queries from [2020](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1228)
I have to run them again to see if we can add anything.

- [x] Adoption of image formats in SSGs
- [x] Core Web Vitals distribution by CDNs hosted top SSGs
- [x] Core Web Vitals of CDNs hosted SSGs
- [x] Distribution of page weight, requests, and co2 grams per SSG web page
- [x] Lighthouse category scores per SSG
- [x] SSG adoptions and top SSGs YoY
- [x] Core Web Vitals distribution by SSG
- [x] Core Web Vitals performance by SSG
- [x] Third party bytes and requests on SSGs
- [x] Jamstack rank magnitude (isn't tested)

### CODE REVIEW

File: sql/2021/jamstack/adoption_of_image_formats_in_ssgs.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    `httparchive.almanac.requests`
+  WHERE
+    date = '2021-08-01' AND
+    type = 'image')
+JOIN (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    url AS page
+  FROM
+    `httparchive.technologies.2021_08_01_*`
```
Comment (rviscomi) [inline sql/2021/jamstack/adoption_of_image_formats_in_ssgs.sql:24]: Be sure to point all 2021 queries at the July dataset
```suggestion
    `httparchive.technologies.2021_07_01_*`
```

**Comment by rviscomi** [comment]
Hi @denar90 just checking in on your progress. Is there anything I can do to help?

**Comment by denar90** [comment]
Testing queries...

**Review by rviscomi — APPROVED** [review]

**Comment by denar90** [comment]
@rviscomi thank you for the review 🎉

**Pull request merged**